cmd/podman/save.go

```diff
@@ -125,7 +126,18 @@ func saveCmd(c *cli.Context) error {
 	dest := dst
 	// need dest to be in the format transport:path:reference for the following transports
 	if (strings.Contains(dst, libpod.OCIArchive) || strings.Contains(dst, libpod.DockerArchive)) && !strings.Contains(newImage.ID(), args[0]) {
-		dest = dst + ":" + args[0]
+		prepend := ""
```

Comment:
style/readability nit.  I'd rather see args[0] be assigned to a variable that's named to the expected content of args[0].  Especially so since you're referencing it several times here.